client/ting.jsx

```diff
@@ -43,9 +44,9 @@ const Ting = React.createClass({
         const URL = window.location.hostname + ':' + config.port;
         this._socket = io.connect(URL, {secure: config.websocket.secure});
 
-        this._socket.on('login-response', ({success, people, error}) => {
+        this._socket.on('login-response', ({success, people, errorUsername, errorPassword}) => {
```

Comment:
This requires arch changes. Please make a separate PR. I recommend we keep the `error` parameter and have various errors there in this form:

username-empty
username-length
username-taken
username-chars
password-empty
password-incorrect